src/auth/Cargo.toml

```diff
@@ -18,7 +18,7 @@ name = "google-cloud-auth"
 #     version of all downstream dependencies. For details see:
 #         https://github.com/googleapis/google-cloud-rust/issues/3237
 #         https://github.com/googleapis/google-cloud-rust/issues/3265
-version     = "1.1.0"
+version     = "1.2.0"
```

Comment:
I would override this to `1.1.1`. No customer visible changes yet.

Reply:
Ack, my mistake.

I set it to `1.1.1`, but maybe we would be better off reverting the bump altogether. Thoughts?

There would be less risk of calling a minor bump a patch bump (if we make a change that requires a minor bump before the release). And we will probably run `rust-bump-versions` before the next release anyway (so I doubt we will miss the bump altogether).